### PR TITLE
Extend troubleshooting

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

Extend troubleshooting guides

**Why**:

We should add as much common issues and their solutions as possible to avoid unnecessary issues for our community users.

**How**:

**Checklist**:

- [ ] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

**Review by y0urself — APPROVED** [review]
Noice!

File: src/common/container/troubleshooting.md [hunk 1]
```diff
@@ -38,3 +38,28 @@ The code behind these errors tries to set up the database. If the database is
 already initialized, all tables, users, permissions and extensions exist,
 errors are raised. At the moment, it is not possible to silence these errors but
 they can be ignored safely.
+
+### `osp_scanner_feed_version: failed to connect to /run/ospd/ospd-openvas.sock`
+
+If the `gvmd` log contains this error `gvmd` is not able to connect to the
```
Comment (k-schlosser) [inline src/common/container/troubleshooting.md:44]:
```suggestion
If the `gvmd` log contains this error, `gvmd` is not able to connect to the
```

File: src/common/container/troubleshooting.md [hunk 2]
```diff
@@ -38,3 +38,28 @@ The code behind these errors tries to set up the database. If the database is
...
+`ospd-openvas` scanner. Most likely `ospd-openvas` isn't running. Try to restart
```
Comment (k-schlosser) [inline src/common/container/troubleshooting.md:45]:
```suggestion
`ospd-openvas` scanner. Most likely `ospd-openvas` is not running. Try to restart
```

File: src/common/container/troubleshooting.md [hunk 3]
```diff
@@ -38,3 +38,28 @@ The code behind these errors tries to set up the database. If the database is
...
+
+```{code-block} shell
+---
+caption: Restart the ospd-openvas scanner
+---
+docker-compose -f $DOWNLOAD_DIR/docker-compose.yml -p greenbone-community-edition \
+    restart ospd-openvas
+```
+
+If you still get errors you need to take a look at the `ospd-openvas` logs.
```
Comment (k-schlosser) [inline src/common/container/troubleshooting.md:57]:
```suggestion
If you still get errors, you need to take a look at the `ospd-openvas` logs.
```

File: src/common/source-build/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+```{warning}
+The commands in the following troubleshooting guidelines are only valid if you
+followed the source build documentation. If you installed the Greenbone
+Community Edition via a Linux Distribution (for example Kali Linux) the commands
```
Comment (k-schlosser) [inline src/common/source-build/troubleshooting.md:4]:
```suggestion
Community Edition via a Linux Distribution (for example Kali Linux), the commands
```

File: src/common/source-build/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+may be slightly different and need to be adjusted.
+```
+
+### Failed to find port_list ‘33d0cd82-57c6-11e1-8ed1-406186ea4fc5‘
+
+This error can occur when tying to use the Task Wizard to create a quick scan
+after the initial setup. It is cause by `gvmd` not being able to load the port
+lists from the feed into its database. To fix this issue you first of all it
+is required to check if `/var/lib/gvm/data-objects/` contains the port list XML
```
Comment (k-schlosser) [inline src/common/source-build/troubleshooting.md:13]:
```suggestion
lists from the feed into its database. To fix this issue, you first need to
check if `/var/lib/gvm/data-objects/` contains the port list XML
```

File: src/common/source-build/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+file:
+
+```{code-block} shell
+---
+caption: Check if port list is already synced
+---
+find /var/lib/gvm/data-objects/ -name "*33d0cd82-57c6-11e1-8ed1-406186ea4fc5*.xml"
+```
+
+If the `find` command doesn't return a XML file for your release the data
```
Comment (k-schlosser) [inline src/common/source-build/troubleshooting.md:23]:
```suggestion
If the `find` command does not return an XML file for your release, the
```

File: src/common/source-build/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+data objects haven't been synced (yet).
```
Comment (k-schlosser) [inline src/common/source-build/troubleshooting.md:24]:
```suggestion
data objects have not been synced (yet).
```

File: src/common/source-build/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+```{code-block} shell
+---
+caption: Syncing data objects processed by gvmd
+---
+sudo -u gvm greenbone-feed-sync --type GVMD_DATA
+```
+
+Second `gvmd` should be forced to (re-)load the data objects including the port
```
Comment (k-schlosser) [inline src/common/source-build/troubleshooting.md:33]:
```suggestion
Second, `gvmd` should be forced to (re-)load the data objects including the port
```

File: src/faq.md [hunk 9]
```diff
@@ -7,13 +7,13 @@ For a comprehensive background see {doc}`History of OpenVAS <history>`.
...
+Besides that, the Greenbone Community can be reached at the [community forum]
 to ask question. In this forum, several Greenbone developers try to help on a
 voluntary basis.
 
 ## Can you help with my issue on Kali, Cent OS, XYZ distribution?
 
-[https://community.greenbone.net/t/openvas-showing-503-error/5465/4](https://community.greenbone.net/t/openvas-showing-503-error/5465/4)
+[https://forum.greenbone.net/t/openvas-showing-503-error/5465/4](https://forum.greenbone.net/t/openvas-showing-503-error/5465/4)
 
 Greenbone doesn't provide any packages for any Linux distribution besides the
```
Comment (k-schlosser) [inline src/faq.md:18]:
```suggestion
Greenbone does not provide any packages for any Linux distribution besides the
```

**Review by k-schlosser — CHANGES_REQUESTED** [review]

**Comment by bjoernricks** [comment]
@k-schlosser I've pushed the proposed review changes to the main branch (bf8860a0457bfff86b08438a058326b7cd220667).

**Pull request merged**